pkg/providers/tinkerbell/hardware/validator.go

```diff
@@ -134,6 +136,17 @@ func StaticMachineAssertions() MachineAssertion {
 			}
 		}
 
+		if m.VLANID != "" {
+			i, err := strconv.Atoi(m.VLANID)
+			if err != nil {
+				return errors.New("VLANID: must be an integer")
+			}
+
+			if i < 1 || i > 4094 {
```

Comment:
Could we throw down a constant name and/or a link to something like https://en.m.wikipedia.org/wiki/VLAN#IEEE_802.1Q so the 4094 is more understandable?

I double took at this and googled it.

Reply:
for sure, will add.

Reply:
added